code/datums/emotes.dm

```diff
@@ -67,7 +67,7 @@
 
 	for(var/i in GLOB.dead_mob_list)
 		var/mob/M = i
-		if(isnewplayer(M) || !M.client)
+		if(isnewplayer(M) || !M?.client)
```

Comment:
chicken on magmoor are runtiming.

Unrelated to this, but this happened to often so i'm fixing it